{
  "errors": [
    {
      "filename": "on_device_embedding_test.py",
      "reason": "No such file or directory"
    }
  ],
  "generated_at": "2021-10-30T04:43:25Z",
  "metrics": {
    "_totals": {
      "CONFIDENCE.HIGH": 0,
      "CONFIDENCE.LOW": 0,
      "CONFIDENCE.MEDIUM": 0,
      "CONFIDENCE.UNDEFINED": 0,
      "SEVERITY.HIGH": 0,
      "SEVERITY.LOW": 0,
      "SEVERITY.MEDIUM": 0,
      "SEVERITY.UNDEFINED": 0,
      "loc": 0,
      "nosec": 0
    }
  },
  "results": []
}